src/services/User.js

```diff
@@ -8,25 +8,19 @@ const postUser = async ({ username, email, password }) => {
 };
 
 const getGuest = async () => {
-    const response = await fetch('/api/users/guest', {
-        method: 'GET',
-        headers: { 'Content-Type': 'application/json' }
-    });
+    const response = await fetch('/api/users/guest');
     return await response.json();
 };
 
 const getMe = async () => {
-    const response = await fetch('/api/users/me', {
-        method: 'GET',
-        headers: { 'Content-Type': 'application/json', credentials: 'include' }
-    });
+    const response = await fetch('/api/users/me');
     return await response.json();
 };
 
 const logoutUser = async () => {
     const response = await fetch('/api/users/logout', {
         method: 'POST',
-        headers: { 'Content-Type': 'application/json', credentials: 'include' }
+        headers: { 'Content-Type': 'application/json' }
```

Comment:
Tikrinai ar veikia? Nes gi sort of esme kad pakeicia cookie expiry time.

Reply:
as irgi buvau amazed kad veikia be jo. Siaip kinda idomu ka jis tiksliai daro.